utils/fabric/Makefile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Intel Corporation
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+TOP = ../..
+include $(TOP)/build.mk
+
+GO_CMDS= get-fabric-container-name
+
+build: ${GO_CMDS}
+
+${GO_CMDS}:
+	$(GO) build $@.go
+
+test: build
+#	$(GO) test -v
```

Comment:
Wouldn't it be nice to have a test here? :)

Reply:
the program is so simple and actual semantics is defined by fabric, so not sure what test program i could meaningfully write.  I'll remove the test from the makefile, though